src/hooks/useRepositoryStorage.js

```diff
@@ -1,32 +1,48 @@
-import { useState, useCallback } from 'react';
-import useAsyncStorage from '../hooks/useAsyncStorage';
-import notifyMessage from '../utils/notifyMessage';
+import useSWR from 'swr';
+import AsyncStorage from '@react-native-async-storage/async-storage';
 import { ASYNC_STORAGE_KEY, STORED_DATA_MAX } from '../constants/repository';
+import notifyMessage from '../utils/notifyMessage';
+
+const getStoredValue = async (key) => {
+  try {
+    const item = await AsyncStorage.getItem(key);
+    return item ? JSON.parse(item) : null;
+  } catch (e) {
+    console.log(e);
+  }
+};
 
-export default function useRepositoryStorage(initialValue) {
-  const [storedValue, setStoredValue] = useAsyncStorage(
-    ASYNC_STORAGE_KEY,
-    initialValue,
-  );
-  const [storedRepo, setStoredRepo] = useState(() => {
-    return storedValue || initialValue;
+function useRepositoryStorage() {
+  const { data, mutate } = useSWR(ASYNC_STORAGE_KEY, () => {
```

Comment:
```suggestion
  const { data, mutate } = useSWR(ASYNC_STORAGE_KEY, getStoredValue);
```
**Recommend-Request**: 여기 argument도 축약이 가능한 것으로 보입니다.